h5c/vic/src/vic-webapp/src/app/configure/configure-vch-modal/security/configure-vch-modal-security.component.html

```diff
@@ -0,0 +1,19 @@
+<ng-container *ngIf="resourceObjRefObs | async as info; else loadingContent">
+  <vic-configure-vch-modal-container [title]="'Security'"
```

Comment:
If you want to pass an string to the internal "@Input() title: string;" property of the "vic-configure-vch-modal-container" component, you have to use the nested quoting, otherwise it will be handled as a variable reference on the host component.

Anyway, it has been replaced with the i18n.